README.md

```diff
@@ -99,9 +99,11 @@ the parser and the printer. To get started, run:
 [homebrew]: https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/shfmt.rb
 [nixos]: https://github.com/NixOS/nixpkgs/blob/HEAD/pkgs/tools/text/shfmt/default.nix
 [void]: https://github.com/voidlinux/void-packages/blob/HEAD/srcpkgs/shfmt/template
+[crux]: https://github.com/6c37/crux-ports-git/tree/3.3/shfmt
```

Comment:
also, won't this link get stale due to the `3.3`? is there no `master` equivalent?

Reply:
no sorry there isn't, I'll rename `3.3` to master next CRUX release.